src/utils/components/ComponentVersionTable.tsx

```diff
@@ -0,0 +1,33 @@
+import Text from "../../components/Text/Text";
+
+export interface ComponentVersionProps {
+  /** The DS version when the component was added*/
+  added: string;
+  /** The DS version with the most recent version of the component */
+  latest: string;
+}
+
+export const ComponentVersion = (props: ComponentVersionProps) => {
```

Comment:
Just a note for consistency: the file has `Table` but the component name does not. Similar comment when importing the component in the other files.

Reply:
Good catch. I added `Table` to the component name last minute, but forgot to have that change cascade throughout all references.